rest/taskana-rest-spring/src/main/java/pro/taskana/classification/rest/ClassificationDefinitionController.java

```diff
@@ -84,6 +84,10 @@
                 Collectors.collectingAndThen(
                     Collectors.toList(), classificationRepresentationModelAssembler::toPageModel));
 
+    pageModel
+        .getContent()
+        .forEach(ClassificationRepresentationModel::removeLinks);
```

Comment:
I would rather expect that we do remove the creation of the links?

Reply:
Me too